src/main/java/com/google/devtools/build/lib/rules/python/PyCommon.java

```diff
@@ -132,9 +133,12 @@ public void initBinary(List<Artifact> srcs) {
     addPyExtraActionPseudoAction();
   }
 
-  public void addCommonTransitiveInfoProviders(RuleConfiguredTargetBuilder builder,
-      PythonSemantics semantics, NestedSet<Artifact> filesToBuild) {
-
+  public void addCommonTransitiveInfoProviders(
+      RuleContext ruleContext,
```

Comment:
I don't exactly remember as this has been really old. I think I had to add it because rule context in pycommon was different from pylibrary/pybinary.